bpfman/src/bin/cli/args.rs

```diff
@@ -155,6 +155,10 @@ pub(crate) enum LoadCommands {
         /// [default: pass, dispatcher_return]
         #[clap(long, verbatim_doc_comment, num_args(1..))]
         proceed_on: Vec<String>,
+
+        /// Optional: The name of the target network namespace.
+        #[clap(short, long)]
+        netns: Option<String>,
```

Comment:
Good suggestion.  This will be better because it handles both cases.  I've also been thinking about the kubernetes api for this.  If an app is attaching a program in a network namespace directly on the node, then it probably can know the path, but if the app needs to attach to an interface in the network namespace of a kubernetes pod/container, the app probably doesn't know the path.  However, I think we can use the existing container selector, and our code can figure it out like we do for uprobes.  So, for kubernetes, I was thinking we'd need to provide the user two options: the path to a netns, or a container selector.  What do you think?

Reply:
The user-facing K8s API should only be selector-based.
bpfman's API will require a path.
bpfman-agent will have to figure out the mapping from container selector -> container ID -> network namespace